src/main/java/com/vmtapp/enterprise/TicketController.java

```diff
@@ -42,19 +45,19 @@ public String saveTicket(Ticket ticket){
     }
 
     @RequestMapping("/ticket")
-    public ResponseEntity fetchAllTickets(){
-        return new ResponseEntity(HttpStatus.OK);
+    public List fetchAllTickets(){
+        return ticketService.fetchAllTickets();
```

Comment:
The new ResponseEntity is not actually returning any messages. You need to write functions to it to make it return all tickets. Source:https://codeblog.jonskeet.uk/2006/06/10/deadlysins/ #3

Reply:
Since we're in a controller, I expect that it would return a ResponseEntity.